src/persist/src/indexed/cache.rs

```diff
@@ -49,9 +49,7 @@ pub struct BlobCache<B> {
     metrics: Arc<Metrics>,
     blob: Arc<Mutex<B>>,
     async_runtime: Arc<AsyncRuntime>,
-    // TODO: Use a disk-backed LRU cache.
-    unsealed: Arc<Mutex<HashMap<String, Arc<BlobUnsealedBatch>>>>,
-    trace: Arc<Mutex<HashMap<String, Arc<BlobTraceBatch>>>>,
+    cache: Arc<BlobCacheInner>,
```

Comment:
BlobCacheInner already has Arc on all of its fields, so having one here also is redundant. probably remove the BlobCacheInner ones?

Reply:
ofc! done